rest/method_get.go

```diff
@@ -58,6 +59,20 @@ func listGet(ctx context.Context, r *http.Request, route *RouteMatch) (status in
 		e = NewError(err)
 		return e.Code, nil, e
 	}
+	// Now we can set Total.
+	list.Total = len(list.Items)
```

Comment:
This is the current page size, not the total. Only the storage handler can know (or not) the total elements in the collection.